vcluster/_fragments/patches.mdx

```diff
@@ -3,38 +3,71 @@ import CodeBlock from '@theme/CodeBlock';
+
+:::info Using wildcards in patches
+You can apply a wildcard, using an asterisk (`*`) in a specified path, to modify all elements of an array or object.
+For instance, `spec.containers[*].name` selects the `name` field of every container in the `spec.containers` array, and `spec.containers[*].volumeMounts[*]` selects all `volumeMounts` for each container. 
+When using the asterisk (`*`) notation, vCluster applies changes individually to every element that matches the path.
 :::
 
-### JavaScript Expression Patches
+### JavaScript expression patch
+
+A JavaScript expression patch allows you to use JavaScript ES6 expressions to change specific fields when syncing between virtual and host clusters. 
+This is useful when modifying resource configurations to align with differing environments or naming conventions between clusters.
+
+For example, if your virtual cluster follows a specific naming convention for container names, but the host cluster requires a different prefix, you can use a JavaScript expression patch to automatically update the relevant field for each container.
+
+You can define a path for `spec.containers[*].name` in `vcluster.yaml` using the following configuration:
 
-These are powerful JavaScript ES6 compatible expression patches that can be used to change a field while syncing. You define how it changes when syncing from the virtual cluster into the host cluster or when syncing from the host cluster into the virtual cluster. To change the path <span>{props.path}</span> you can do:
 <CodeBlock language="yaml">
-{`sync:
+{`
+sync:
   toHost:
-    ${props.resource}:
-      enabled: true
-      patches:
-      - path: ${props.path}
-        expression: '"my-prefix-"+value'
-        # optional reverseExpression to reverse the change from the host cluster
-        # reverseExpression: 'value.slice("my-prefix".length)'`}
+    enabled: true
+    patches:
+    - path: spec.containers[*].name
+      expression: "'my-prefix-' + value"
+`}
 </CodeBlock>
 
-There is also a variable called `context` besides `value` that can be used to access specific data of the virtual cluster:
-* `context.vcluster.name`: Name of the virtual cluster
-* `context.vcluster.namespace`: Namespace of the virtual cluster
-* `context.vcluster.config`: Config of the virtual cluster, basically `vcluster.yaml` merged with the defaults
-* `context.hostObject`: Host object (can be null if not available)
-* `context.virtualObject`: Virtual object (can be null if not available)
-* `context.path`: The matched path on the object, useful when using wildcard path selectors (*)
+In the example:
+
+  - `spec.containers[*].name` targets the `name` field of all containers within a Pod. The `*` wildcard applies the patch to each container individually.
```

Comment:
Since this fragment is reused throughout all the different resources and the path changes,
 Instead of referencing `Pod` directly, you should use `${props.resource}`. 
 
 Since the `path` of the resource also changes dynamically, it is hard to templatize being able to explain the example. What we can try to do is
 
 ```
 The path identifies the field in ${props.resource} that you want to override when syncing to the host cluster. The expression is a JavaScript expression that tells you how to override the field. In this case, `"my-prefix-"` will be prepended to the field of the ${props.resource} in the host cluster. 
```